### ISSUE TITLE
[Discussion]: Update Old Patches to 4.0 Standards

### ISSUE THREAD

**Issue by Breadbox2k19** [CombatExtended-Continued/CombatExtended#821]
# Update motes into Flecks
Sometimes, custom damagedefs(which still use the old motes rather than flecks) are created for the purpose of convenience.
-Typically found in weapons and animals patches.

# Outside Squishy

Goes without saying, bodies of all animals and pawns(or at least all the pawn), should be patched so that small bodyparts eyes, noses and jaws no longer absorb high damage projectiles outright.
Tostov pls help me with the anime races......

# Night vision Update

Races and animals with inherent night vision (Nightlings, Forsakens etc etc) should be patched to have innate night sight. I think some mechs/machines added by mods also need to have night vision added since they do not inherit from mechanoidbase.
All power armour helmets, shooting enhancing headgears and optical implants would also need to be updated. Quite a lot of work.

# initialResistanceRange and initialWillRange
Introduced in 1.3, no idea what either of these do, but they need to be added for any pawns added via patching.

# Traders Update
Need to add heavy ammo to traders added by mods.

# Add Grenades and stuff loadout of modded factions.
Basically needs to be done for all industrial or later factions.

# External Patches

There are a few patches maintained externally that I would like to move to CE proper, they tend to be pretty terribly done and never receive any updates.
